cmd/image-builder-migrate-db-tern/image-builder-migrate-db-tern.go

```diff
@@ -90,14 +88,11 @@ func main() {
 		"--sslmode", conf.PGSSLMode)
 	out, err := cmd.CombinedOutput()
 
-	scanner := bufio.NewScanner(bytes.NewReader(out))
-	for scanner.Scan() {
-		slog.InfoContext(ctx, scanner.Text())
-	}
-
 	if err != nil {
+		slog.ErrorContext(ctx, "migration output", "output", string(out))
 		panic(err)
 	}
 
+	slog.InfoContext(ctx, "migration output", "output", string(out))
```

Comment:
The linter helped me to fix this one, messages should really be constants, data should always go into structured fields.